src/Recaptcha.js

```diff
@@ -50,9 +51,10 @@ class Recaptcha extends React.Component {
   render() {
     const { className, sitekey, ...otherProps } = this.props;
 
+
     return (
       <div
-        {...otherProps}
+        {...omit(otherProps, Object.keys(this.constructor.propTypes)) }
```

Comment:
I think there will be a problem if consumers remove `propTypes` in their project by [babel-plugin-transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types). (Actually I use this babel-plugin in all my projects.)

How about  this:

```
{...omit(otherProps, ['props', 'to', 'omit']) }
```